trio/_core/_run.py

```diff
@@ -1805,10 +1805,57 @@ def start_guest_run(
     run_sync_soon_threadsafe,
     done_callback,
     run_sync_soon_not_threadsafe=None,
-    trust_host_loop_to_wake_on_signals=False,
+    host_uses_signal_set_wakeup_fd=False,
     clock=None,
     instruments=(),
 ):
+    """Start a "guest" run of Trio on top of some other "host" event loop.
+
+    Each host loop can only have one guest run at a time.
+
+    You should always let the Trio run finish before stopping the host loop;
+    if not, it may leave Trio's internal data structures in an inconsistent
+    state. You might be able to get away with it if you immediately exit the
+    program, but it's safest not to go there in the first place.
+
+    So generally, the best way to do this is wrap this in a function that
+    starts the host loop and then immediately starts the guest run, and then
+    shuts down the host when the guest run completes.
+
+    Args:
+
+      run_sync_soon_threadsafe: An arbitrary callable, which will be passed a
+         function as its sole argument::
+
+            def my_run_sync_soon_threadsafe(fn):
+                hi()
```

Comment:
Maybe just `...` as the body? I'm not sure `hi()` adds any information

Reply:
Ha, whoops! I had `...`, but then "temporarily" swapped it while trying to track down a weird sphinx error about not being able to tokenize the code bock... thank you, that would have been embarrassing :-)